exporter/v1_compatibility.go

```diff
@@ -59,10 +54,10 @@ type conversion struct {
 	suffixMapping    map[string]string
 }
 
-func rawToCompatibleRawMetric(rm *rawMetric) *rawMetric {
+func metricRenameAndLabel(rm *rawMetric, convs []conversion) *rawMetric {
```

Comment:
it should be moved outside of this package since we use it not only for compatibility mode.

Reply:
Done. Moved to metrics.go